app/models/edition_lead_image.rb

```diff
@@ -1,4 +1,6 @@
 class EditionLeadImage < ApplicationRecord
   belongs_to :edition
   belongs_to :image
+
+  accepts_nested_attributes_for :edition
```

Comment:
This is not a very useful comment, but I'd just like to say that this thing gives me the heebies. It's just so incredibly magical

Reply:
yup it's the worst. We can pull a lot of this out once we move to form objects, but until we have a way of pulling the validations together and not relying on the updater pub/sub service i think we're stuck with it 😞